client/src/styles/components/drop-down/drop-down.module.scss

```
@@ -0,0 +1,48 @@
+@import '@/styles/theme.scss';
+
+.drop-down {
+  display: flex;
+  flex-direction: column;
+
+  width: 165px;
+
+  box-shadow: 0px 2px 4px 0px #00000040;
+
+  border-radius: 10px 10px 10px 10px;
+}
+
+.item {
+  display: flex;
+
+  align-items: center;
+
+  padding: 16px;
+
+  background-color: $color-off-white;
+  color: $color-title-active;
+
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 16px;
+
+  width: 100%;
+  height: 48px;
+
+  border-bottom: 1px solid #00000040;
+
+  &.normal {
+    color: $color-title-active;
+  }
+
+  &.highlighted {
+    color: $color-red;
+  }
+
+  &.first {
+    border-radius: 10px 10px 0px 0px;
+  }
+
+  &.last {
+    border-radius: 0px 0px 10px 10px;
+  }
+}
```

```suggestion

  &:hover {
    background-color: $color-grey3;
  }
}
```

:hover를 만들면 어떨까 합니다. 중요한건 아니고 그냥 이팩트에요 ㅎㅎ

---

좋습니당 ㅎㅎ

---

훨씬 좋네요👍👍